pages/configuration/elasticsearch.rst

```diff
@@ -149,7 +118,7 @@ Tuning Elasticsearch
 
 Graylog is already setting specific configuration for every index it is managing. This is enough tuning for a lot of use cases and setups.
 
-A more detailed guide about tuning Elasticsearch will be published at a later time.
+More detailed information about the configuration of Elasticsearch can be found in the `official documentation <https://www.elastic.co/guide/en/elasticsearch/reference/current/system-config.html>`__.
```

Comment:
This should be a link to the specific version of the Elasticsearch documentation:
https://www.elastic.co/guide/en/elasticsearch/reference/5.4/system-config.html